Lib9c/Action/UpdateSell.cs

```diff
@@ -23,11 +23,25 @@ namespace Nekoyume.Action
     /// Updated at https://github.com/planetarium/lib9c/pull/1022
     /// </summary>
     [Serializable]
-    [ActionType("update_sell3")]
+    [ActionType("update_sell4")]
```

Comment:
Obsoletion may be needed. 🤔

Reply:
obsolete index set on update_sell3 action